theme/jade/index-partials/header.jade

```diff
@@ -1,9 +1,9 @@
 header.main-header
   img.main-header__logo(src='images/jsunconflogo.svg', alt='JSUnconf')
 
-  p.main-header__title
+  h3.main-header__title
     | Hamburg, 23rd &amp; 24th of April 2016
-  p.main-header__tagline
+  h2.main-header__tagline
```

Comment:
H1 would be the title of the page – the logo. Can add a H1 for that. There should always only be one. For the date – I thought it more as a subheading (see the class), so it is a heading. Can revert this, though.